extensions/xjc-plugins/runtime/src/main/doc/intro.adoc

```diff
@@ -1,12 +1,12 @@
-XJC plugins for `wsdl2java` code generation.
+XJC plugins for xref:user-guide/generate-java-from-wsdl.adoc[wsdl2java] code generation.
 You'll need to add this extension if you want to use any of the following in
 xref:reference/extensions/quarkus-cxf.adoc#quarkus-cxf_quarkus.cxf.codegen.wsdl2java.additional-params[quarkus.cxf.codegen.wsdl2java.additional-params]:
 
-* `-xjc-Xboolean`
-* `-xjc-Xdv`
-* `-xjc-Xjavadoc`
-* `-xjc-Xpl`
-* `-xjc-Xts`
-* `-xjc-Xwsdlextension`
+* `https://cxf.apache.org/cxf-xjc-boolean.html[-xjc-Xbg]` - generate `getFoo()` instead of `isFoo()` accessor methods for boolean fields.
+* `https://cxf.apache.org/cxf-xjc-dv.html[-xjc-Xdv]` - let the generated getter methods return the default value defined in the schema unless the field is set explicitly.
+* `-xjc-Xjavadoc` - generate JavaDoc based on `xs:documentation` present in the schema.
+* `-xjc-Xproperty-listener` - add `PropertyChangeListener` support to the generated beans.
+* `https://cxf.apache.org/cxf-xjc-ts.html[-xjc-Xts]` - generate `toString()` methods in model classes.
+* `https://cxf.apache.org/cxf-xjc-wsdlextension.html[-xjc-Xwsdlextension]` - generate beans that can be used directly with WSDL4J as extensors in the WSDL.
```

Comment:
@llowinge this is the main docs change. As you can see I have not found suitable CXF pages for all options. 
Some of the options were bogus, perhaps some old names that do not work now. I better added a test below to make sure that the spelling of the option names is correct. In other words, thanks for complaining @llowinge!